thonny/plugins/files.py

```diff
@@ -132,11 +132,15 @@ def __init__(self, master):
         get_workbench().bind("ToplevelResponse", self.on_toplevel_response, True)
 
         self.ctxdisp = FileBrowserContextDispatcher(self)
-        self.ctxdisp.add_handler(VirtualEnvContextHandler())
-        self.ctxdisp.add_handler(ShellScriptContextHandler())
-        self.ctxdisp.add_handler(PythonShellScriptContextHandler())
-        self.ctxdisp.add_handler(RequirementsFreezeContextHandler())
-        self.ctxdisp.add_handler(RequirementsInstallerContextHandler())
+        self.add_ctx_handler(VirtualEnvContextHandler())
+        self.add_ctx_handler(ShellScriptContextHandler())
+        self.add_ctx_handler(PythonShellScriptContextHandler())
+        self.add_ctx_handler(RequirementsFreezeContextHandler())
+        self.add_ctx_handler(RequirementsInstallerContextHandler())
+        self.add_ctx_handler(UnittestContextHandler())
+
+    def add_ctx_handler(self, handler):
```

Comment:
intended to use by other plugins

Reply:
todo:
make this available via e.g. `get_workbench().add_filebrowser_handler(handler)`